### PR TITLE
Feature/section landing image banner

### PR DESCRIPTION
**Pull request by netdesignr**
Hi @sdwilkes,

Can you please do the PR for research and redesign new template?

Thank you!

Kindest regards,
Mihai

### CODE REVIEW

**Review by sdwilkes — APPROVED** [review]
All looks good @netdesignr . I've tested it locally, no issues found.

EDIT - just one small issue with spacing/padding found (see below)

**Comment by sdwilkes** [comment]
@grange85 Hi Andy, can we put feature/section_landing_image_banner on TEST so we can test the changes? It's been tested locally but will be good do some extra testing with real content

File: inc/functions/functions-global.php [hunk 1]
```diff
@@ -16,6 +16,75 @@ function tna_child_styles()
...
+        && function_exists('the_content')
+    ) {
+        $buttonTitle = get_post_meta($post->ID, 'action_button_title', true);
+        $buttonUrl = get_post_meta($post->ID, 'action_button_url', true);
+        if (empty($post->post_content)) {
+            // Do nothing - banner content overlay is not displayed
+        } else { // Banner content overlay ?>
+            <div class="entry-content">
+                <div class="col-xs-9 page-content">
+                    <?php the_content(); ?>
```
Comment (sdwilkes) [inline inc/functions/functions-global.php:77]: @netdesignr Add 10px of padding to the top of the paragraph to fix spacing issue

Comment (netdesignr) [reply inc/functions/functions-global.php:77]: Done

Comment (sdwilkes) [reply inc/functions/functions-global.php:77]: Thanks @netdesignr , have checked these changes and they're all good.

**Pull request merged**